app/controllers/hyrax/admin/admin_sets_controller.rb

```diff
@@ -106,7 +106,7 @@ def create_admin_set
       def setup_form
         add_breadcrumb t(:'hyrax.controls.home'), root_path
         add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.admin'), hyrax.dashboard_path
-        add_breadcrumb t(:'hyrax.admin.sidebar.admin_sets'), hyrax.admin_admin_sets_path
+        add_breadcrumb t(:'hyrax.dashboard.my.action.edit_admin_set'), hyrax.edit_admin_admin_set_path(:id)
```

Comment:
Does this draw the correct link?  I'm not sure you can pass a symbol as the first argument.  I'm also wondering if this is the correct behavior for the `new` action, which also calls this method.

Reply:
Thanks @jcoyne I'll push up a fix sometime today.